templates/metrics/css/_css_metric_nav.html

```diff
@@ -10,25 +10,33 @@ <h3>Animated properties</h3>
 
 <!-- For Narror ViewPorts with small screen size -->
 <div id="horizontal-sub-nav">
-  <table>
-    <tr>
-      <nav class="data-panel">
```

Comment:
The html validation complained about the nav tag in the table, mentioning a voodoo mode.  So I moved the nav elements outside the table, and created a second table for the second part.  This means the columns won't be aligned, but otherwise looks ok:

<nav class="data-panel" style="box-sizing: border-box; list-style: none; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-variant-numeric: inherit; font-variant-east-asian: inherit; font-weight: 400; font-stretch: inherit; font-size: 14px; line-height: inherit; font-family: Roboto, sans-serif; text-decoration-line: inherit; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial; -webkit-tap-highlight-color: rgba(0, 0, 0, 0); max-width: 760px; margin: 0px 10px; color: rgba(0, 0, 0, 0.87); letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(250, 250, 250);">

View All Properties As: | Timeline \| Stack Rank
-- | --

```
View All Properties As:	Timeline | Stack Rank
View Animated Properties As:	Timeline | Stack Rank
```

Reply:
That's fine.  I think that it is worth it.